<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Navigation</class>
 <widget class="QWidget" name="Navigation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>92</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navigation</string>
  </property>
  <widget class="Marble::ArrowDiscWidget" name="arrowDisc" native="true">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>85</y>
     <width>36</width>
     <height>152</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QToolButton" name="zoomInButton">
      <property name="toolTip">
       <string>Zoom In</string>
      </property>
      <property name="whatsThis">
       <string>Zoom in by pressing this button to see more detail.</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../../marble.qrc">
        <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
      </property>
      <property name="shortcut">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="zoomSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Zoom Slider</string>
      </property>
      <property name="whatsThis">
       <string>Use this slider to adjust the zoom level of the map.</string>
      </property>
      <property name="minimum">
       <number>900</number>
      </property>
      <property name="maximum">
       <number>2400</number>
      </property>
      <property name="value">
       <number>1050</number>
      </property>
      <property name="sliderPosition">
       <number>1050</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="zoomOutButton">
      <property name="toolTip">
       <string>Zoom Out</string>
      </property>
      <property name="whatsThis">
       <string>Zoom out by pressing this button to see less detail.</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../../marble.qrc">
        <normaloff>:/icons/zoom-out.png</normaloff>:/icons/zoom-out.png</iconset>
      </property>
      <property name="shortcut">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Marble::ArrowDiscWidget</class>
   <extends>QWidget</extends>
   <header>ArrowDiscWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../marble.qrc"/>
 </resources>
 <connections/>
</ui>
